inst/rmarkdown/templates/trb/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,97 @@
+    note: Corresponding Author
+    position: Graduate Student
+    email: alice@example.com
+    affiliation: Some Institute of Technology
+  - name: Bob Security
+    position: Assistant Professor
+    email: bob@example.com
+    affiliation: State University
+  - name: Derek Zoolander
+    position: Professor
+    email: alice@example.com
+    affiliation: Some Institute of Technology
+runningheader: "Anonymous, Security, and Zoolander"
+abstract: |
+  The Transportation Research Board (TRB) has unique and seemingly arbitrary requirements for manuscripts submitted for review. These requirements make it difficult to write the manuscripts quickly, and no existing \LaTeX\ style comes close to fooling the guidelines. This represents an initial effort at creating a template to meet the requirements of TRB authors using \LaTeX, R, Sweave, and/or other literate programming software.
+  
+keywords: ["Transportation", "Travel Behavior"]
+bibliography: mybibfile.bib
+biblio-style: unsrtnat
+linenumbers: true
+#numbersections: true
+output: rticles::trb_article
+wordcount: 1176
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+Transportation Research Board Annual Meetings
+==========================
+
+TRB has greatly simplified the submissions to its system, meaning that this
+template is no longer strictly required. However, there is a typical format
+that most people still use by default, and this template implements that change.
+
+The automatic word count is not yet engaged in the rticles version of the
+template. Set the word count using the yaml header. You can reference figures
+with \@ref(fig:figure-example).
+
+
+
+\subsection{History}
+David Pritchard posted the original versions of this template in 2009 and updated it in 2011, soon after TRB began allowing PDF submissions. Gregory Macfarlane made significant adaptations to it in March 2012, allowing for Sweave integration and automatic word and table counts. Ross Wang automated the total word count and made some formatting modifications in July 2015. Version 2.1.1 has been made available on GitHub in January, 2016.  Version 3.1 has been made available on Github (\url{https://github.com/chiehrosswang/TRB_LaTeX_rnw}) in June, 2017.  Versions 2.1.1 Lite and 3.1 Lite were made available on GitHub (\url{https://github.com/chiehrosswang/TRB_LaTeX_tex}) in June, 2017 for users who do not need R and Sweave functions provided in the original verions.  
+Gregory Macfarlane created the `rticles` template in 2021.
+
+# Features
+The template has a number of features that enable quick and painless manuscript authoring.
+
+## Title Page
+The standard \LaTeX `\maketitle` command is not very versatile, so we have replaced it with a `verb1titlepage` environment. This means that the writers will be required to manually enter spacings based on the number of contributors, but the current settings (12pt between authors, 36pt before, and 60pt after them) seems to work well. 
```

Comment:
Just curious: where is it done as I don't see `verb1titlepage` in tex template ? in the cls file ? 
Thanks!